File: old-0.0/struct/src/kotlin/struct.kt
fun main() {
    val animal = Animal(name = "Max", weight = 5.0)
    println(animal.name)
    println(animal.weight)
}

class Animal(name: String, weight: Double) {
    var name: String
    var weight: Double
    init {
        this.name = name
        this.weight = weight
    }
}

File: old-0.0/struct/src/swift/struct.swift
class Animal {
    var name: String
    var weight: Float64
    init(name: String, weight: Float64) {
        self.name = name
        self.weight = weight
    }
}

let animal = Animal(name: "Max", weight: 5)
print(animal.name)
print(animal.weight)